.github/workflows/release-on-merge.yml

```diff
@@ -55,14 +55,12 @@ jobs:
             https://api.github.com/repos/lightdash/helm-charts/action/workflows/increment-lightdash-app-version.yml/dispatches
       - name: Trigger Sentry release
         if: steps.semantic.outputs.new_release_published == 'true'
+        env:
+          GH_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
         run: |
-          curl -X POST \
+          gh api --method POST \
             -H "Accept: application/vnd.github+json" \
-            -H "Authorization: token ${{ secrets.CI_GITHUB_TOKEN }}" \
-            -d '{
-                  "ref": "main",
-                  "inputs": {
-                    "version": "${{ steps.semantic.outputs.new_release_version }}",
-                  }
-                }' \
-            https://api.github.com/repos/lightdash/lightdash/actions/workflows/create-sentry-release.yml/dispatches
\ No newline at end of file
+            -H "X-GitHub-Api-Version: 2022-11-28" \
```

Comment:
This API version is probably not needed, but just need to check if all works well first! This was taken from this example: https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event